.github/workflows/build.yml

```diff
@@ -94,6 +97,9 @@ jobs:
           if [ "${{ matrix.name }}" = "release" ]; then
             cargo test --release
           fi
+          if [ "${{ matrix.name }}" = "dynamic" ]; then
+            cargo test --features dynamic
```

Comment:
maybe one more idea that can be put in place later in follow-up PR or in long term is to run this on different OS/versions, not just Fedora as this is no longer staticaly compiled from "known-to-be-good" openssl version.